trace/kafka/kafka.go

```diff
@@ -5,27 +5,30 @@ import (
 
 	"github.com/Shopify/sarama"
 	"github.com/beatlabs/patron/correlation"
+	"github.com/beatlabs/patron/encoding"
```

Comment:
Check you IDE for import formatting...

Reply:
I deleted all imports and ran goimports/gofmt from scratch, still provides the same ordering. What are you all using?

Reply:
goimport is the default in vscode, in goland you have to set this via file watchers...

Reply:
I'm using vscode with goimports. Re-run everything, should be okay now.